msal/tests/dev apps/NetCoreTestApp/Program.cs

```diff
@@ -41,15 +41,20 @@ public class Program
         private readonly static string ClientIdForPublicApp = "0615b6ca-88d4-4884-8729-b178178f7c27";
         private readonly static string ClientIdForConfidentialApp = "<enter id>";
 
-        private readonly static string Username = ""; // used for WIA and U/P, can be empty
+        private readonly static string Username = "bogavril@microsoft.com"; // used for WIA and U/P, cannot be empty on .net core
```

Comment:
>"bogavril@microsoft.com" [](start = 50, length = 24)

perhaps you do not want to commit this one

Reply:
Yes, good point :)